akka-stream/src/main/scala/akka/stream/impl/fusing/GraphInterpreter.scala

```diff
@@ -306,7 +306,7 @@ import akka.stream.stage._
         logic.preStart()
       } catch {
         case NonFatal(e) =>
-          log.error(e, "Error during preStart in [{}]: {}", logic.originalStage.getOrElse(logic), e.getMessage)
```

Comment:
not sure why we ever thought that it was better to log the GraphStage rather than the actual logic here, feels like this is much better coming from the logic allowing a custom toString on it potentially provide logic state etc.